### ISSUE TITLE
Better understand where services create accessibility issues when using Design System styles, components and patterns

### ISSUE THREAD

**Issue by kellylee-gds** [alphagov/govuk-design-system-user-needs#80]
## Goals
We know that services which use the Design System can unintentionally create accessibility and usability issues when implementing or iterating our styles, components and patterns

We'd like to learn more about how and why this happens.

## Users

- GOV.UK services
- External accessibility auditors

## Notes from team

> Ideas:
> - Ask the accessibility monitoring team what issues come up a lot across websites using the Design System
> - Ask DAC and other companies used by UK government to do audits what issues come up a lot
> - Review accessibility statements for services to see what issues are listed
> - Ask service teams to submit their audit reports to us (using [GOV.UK](http://gov.uk/) Forms?), and do some analysis on them
> - Look at axe violations recorded in the Service Explorer
> - HMRC might have written a blog post on most common fails? Do they have data they can share
> - Other resources with most common fails
> - [Accessibility monitoring of public sector websites and mobile apps from 2022 to 2024](https://www.gov.uk/government/publications/accessibility-monitoring-of-public-sector-websites-and-mobile-apps-from-2022-to-2024/accessibility-monitoring-of-public-sector-websites-and-mobile-apps-from-2022-to-2024#detailed-tests)
> -[Common accessibility issues in DfE](https://accessibility.education.gov.uk/knowledge-hub/common-issues)
> - [Common WCAG failures (Craig Abbott)](https://www.craigabbott.co.uk/decks/common-wcag-failures/)
> - [Top Ten Most Common Web Accessibility Issues (TPGi)](https://www.tpgi.com/ten-common-web-accessibility-issues/)

> Activities:
> - Get information (using one or more of the above ideas)
> - Analyse and summarise what we’ve learned
> - Play back to the team
> - Ideation workshop for things we might want to do to try and address the common issues

> Who needs to be involved:
> - At a minimum, just Anika, plus the team to be involved in the playback and ideation
> - But there are plenty of opportunities to get other people involved in this work, making accessibility more of a team-owned thing.

> Who we need to actually address any of the common issues will come out of the ideation. There might be some things we want to do user research on, if it’s unclear why the issues are occurring.

> In case we forget, we talked about another potential opportunity to stop the issues occurring:
> - Update [Pattern Checker](https://chromewebstore.google.com/detail/pattern-checker/amjjliajblignodfdjalnfkekkeflkph) or build a similar extension to flag identified issues
> - Point people to Pattern Checker – add to resources and tools page
